config/application.yml.default

```diff
@@ -487,7 +488,6 @@ production:
   password_pepper:
   phone_recaptcha_mock_validator: false
   piv_cac_verify_token_secret:
-  platform_auth_set_up_enabled: false
```

Comment:
Do we want to move this from the `production` section?

Reply:
Never mind, I see that the default is `false` so this line was redundant.

Reply:
Yeah, little bit of clean-up 😄